samples/bluetooth/direct_test_mode/src/dtm.c

```diff
@@ -31,6 +35,10 @@
 #error "No PPI or DPPI"
 #endif
 
+/* Radio fast ramp up time in us. */
+#define FAST_RAMP_UP_TIME 40
+/* Radio normal ramp up time in us */
+#define RAMP_UP_TIME 130
```

Comment:
I checked mostly all our SoC PS. It seems like these values are common for all SoC

Reply:
The ramp-up time depends on the PHY used, and have slight variances in the SoCs. The Zephyr controller has defines for all of these if we need precise values.

Reply:
We could add a include directory using:
`zephyr_include_directories($ENV{ZEPHYR_BASE}/subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio)` and then it would be possible to use for example: `hal_radio_tx_ready_delay_us_get` function from zephyr controller. What do you think about it ?

Reply:
Ok. I added a separated ramp-up time definitions for every PHY based on the value from BLE Controller. The differences between SoCs are small enough to be omitted.